packages/components/src/form-token-field/style.scss

```diff
@@ -48,7 +48,9 @@
 }
 
 .components-form-token-field__help {
-	font-style: italic;
+	font-size: $default-font-size - 1px;
```

Comment:
Should this be using the new `$helptext-font-size` as well?

Reply:
Ah yes! I think I also found another instance. @aaroncampbell would you be able to do another code review, please? Fingers crossed I've caught all those pesky variables now. Thanks so much.